gridstatus/spp.py

```diff
@@ -782,55 +785,178 @@ def get_interconnection_queue(self, verbose=False):
 
         return queue
 
-    @lmp_config(
-        supports={
-            Markets.REAL_TIME_5_MIN: ["latest", "today", "historical"],
-            Markets.DAY_AHEAD_HOURLY: ["latest", "today", "historical"],
-        },
-    )
+    @support_date_range(frequency=None)
```

Comment:
I set the frequency to `None` because with `DAY_START` you can't pass in two times on the same date. In that case, the decorator would set `date` to the beginning of the day and set `end` to `None` which is not what you want.

Reply:
Nevermind this, I updated to use `"5_MINUTE"` frequency.